analysis/featuresDump.csv

```diff
@@ -1,101 +0,0 @@
- id, wordCount, stemmedCount, stemmedEndNN, CD, NN, NNP, NNPS, NNS, PRP, VBG, VBZ, startTuple0, endTuple0, endTuple1, endTuple2, verbBeforeNoun, qMark, qVerbCombo, qTripleScore, sTripleScore, class
```

Comment:
Why has this file been removed?

Reply:
I just followed what @janakrajchadha commented for resolving the conflict, I will try to change this.

Reply:
How did the rebase with the upstream branch delete a file present in it? What I've shared with you is a way to sync your branch with the upstream remote. Not sure if that can lead to this.

Reply:
@janakrajchadha I have just followed those steps and tried to resolve the conflicts but I think I did something wrong but I don't know where, because when I was running those lines they didn't show any error.

Reply:
Restore this file